tests/shared/src/test/scala/scala/meta/tests/parsers/TypeSuite.scala

```diff
@@ -159,4 +159,11 @@ class TypeSuite extends ParseSuite {
     assertNoDiff(exceptionScala2.shortMessage, "illegal literal type (), use Unit instead")
 
   }
+
+  test("plus-minus-then-undescore-source3") {
+    val Type.Function(List(Type.Name("+_")), Type.Name("Int")) =
+      tpe("+_ => Int")(dialects.Scala213Source3)
+    val Type.Apply(Type.Name("Option"), List(Type.Name("-_"))) =
+      tpe("Option[- _]")(dialects.Scala213Source3)
```

Comment:
what about scala3, not just scala2source3?

Reply:
We usually just test the one dialect that it's enabled in.

Reply:
but we are also enabling it in scala 3, which should be tested or removed